contributors/design-proposals/storage/dynamic_volume_limit.md

```diff
@@ -0,0 +1,119 @@
+# Dynamic attached volume limits
+
+## Goals
+
+Currently the number of volumes attachable to a node is either hard-coded or only configurable via an environment variable. Also
```

Comment:
can you list down the those methods here for reference, i didnt know the limit existed somewhere